doc/source/reference/index.rst

```diff
@@ -13,7 +13,9 @@ code.
    code_support
    command-line
    api/api
+   api/modules
```

Comment:
I have a suspicion this was a deliberate exclusion, but I can't back that up.

Reply:
the file is generated, it's not clear to me why we would create it but not include it in the index